scripts/west_commands/sign.py

```diff
@@ -58,7 +57,18 @@
 the build directory using .config and the device tree. A default
 version number of 0.0.0+0 is used (which can be overridden by passing
 "--version x.y.z+w" after "--key"). As shown above, extra arguments
-after a '--' are passed to imgtool directly.'''
+after a '--' are passed to imgtool directly.
+
+rimage
+------
+
+To create a signed binary with the rimage tool, run this from your build
+directory:
+
+   west sign -t rimage -- -k YOUR_SIGNING_KEY.pem
+
+For this to work, either rimage must be installed or you must pass
+the path to rimage using the -p option.'''
```

Comment:
Well, I guess `rimage` must always be installed.
Main question is whether `rimage` is in path.
```suggestion
For this to work, rimage must be installed and in $PATH.
If rimage is installed, but not in $PATH then you must pass
the path to rimage using the -p option.'''
```

Reply:
@tejlmand, I already reworded those test according to @mbolivar suggestions (resolved above). Moreover, it should be consistent with the other tool which has following text:

https://github.com/zephyrproject-rtos/zephyr/blob/4ef29b34e3a29d7b77fddf7baf09990145ab1fed/scripts/west_commands/sign.py#L54-L56

I suggest to change those two places after this path (if needed).

Reply:
> I suggest to change those two places after this path (if needed).

Agree, can we get this in for now and revisit later, since it's already consistent with imgtool?

Reply:
Accepted.